### ISSUE TITLE
Vim-mode's block cursor doesn't work

### ISSUE THREAD

**Issue by calebmeyer** [Matthew-Smith/atom-glowing-cursor#2]
<img width="576" alt="insert" src="https://cloud.githubusercontent.com/assets/3342777/14572732/0187a22e-0316-11e6-9849-5acba487e5f3.png">
<img width="574" alt="normal" src="https://cloud.githubusercontent.com/assets/3342777/14572734/039eacce-0316-11e6-9fc7-377b85da5acd.png">

<img width="803" alt="screen shot 2016-04-15 at 2 27 43 pm" src="https://cloud.githubusercontent.com/assets/3342777/14572761/47039a74-0316-11e6-93b5-013899ff6376.png">
<img width="786" alt="screen shot 2016-04-15 at 2 27 30 pm" src="https://cloud.githubusercontent.com/assets/3342777/14572763/4980678c-0316-11e6-8065-ea78670e56e8.png">

Is there any way we can make block cursors work?

**Comment by keithjgrant** [comment]
Agreed. I love the idea of this plugin, but this makes in a non-starter for me

**Comment by liuchangxing** [comment]
Vim-mode's block cursor doesn't work

**Comment by liuchangxing** [comment]
![qq 20160923131511](https://cloud.githubusercontent.com/assets/22342099/18774855/ca6e172a-818f-11e6-90a5-7b89c1f17308.png)
No use
